src/corporacreator/preprocessors/fr.py

```diff
@@ -1,3 +1,38 @@
+import re
+
+from corporacreator.utils import maybe_normalize, replace_numbers, FIND_PUNCTUATIONS_REG, FIND_MULTIPLE_SPACES_REG
+
+FIND_ORDINAL_REG = re.compile(r"(\d+)([ème|éme|ieme|ier|iere]+)")
```

Comment:
@nicolaspanel Maybe we shoud include potential spaces? I saw data like "1 er".

Reply:
@lissyx since their is no such case in `clips.tsv` I suggest to wait

Reply:
@lissyx ok for you ?

Reply:
Yep.